server/index.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const path = require('path');
+const {
+  sequelize: {
+    models: {
+      Product, Shop, Review, Reviewer, ReviewPhoto,
+    },
+  },
+} = require('../db');
+
+const app = express();
+const PORT = 4000;
+
+// Access components in build.js
+app.use(express.static(path.join(__dirname, '/../public/dist')));
```

Comment:
Took a quick look at your file structure, and it looks like your index.html is not in /public/dist currently. You could statically serve the client folder it is in as well, or put the build and index.html in the same folder and just serve that. Either way, I think you'll need to html file base to add your components to and have that easy access to the build.

Reply:
My assumption was that we won't be needing index.html for our proxy server. So I didn't do this. I just had it in my client folder just for testing purposes.

I probably will move it to my public/dist folder though because thinking about it now, developers using my service could test out what it looks like by going to whatever localhost port they are serving it on.